bin/eslint.js

```diff
@@ -66,6 +67,13 @@ function buildIssueJson(message, path) {
     },
     remediation_points: checks.remediationPoints(checkName, message, cli.getConfigForFile(path))
   };
+
+  var fingerprint = Fingerprint(path, checkName, message.message).compute();
```

Comment:
Shouldn't this use `new`?

Alternatively you could just `export` the `compute` function (which seems to be the prevailing pattern in this repo).

```js
var computeFingerprint = require("../lib/compute_fingerprint");

var fingerprint = computeFingerprint(path, checkName, message.message);
```

The implementation could certainly use a constructor and object internally still.